3-deployment-targets-adapters-and-stacks/bee-rich/package.json

```
@@ -8,7 +8,7 @@
     "build": "run-s \"build:*\"",
     "build:css": "npm run generate:css -- --minify",
     "build:remix": "remix build",
-    "dev": "remix build && run-p \"dev:*\"",
+    "dev": "run-p \"dev:*\"",
```

Both solutions probably work just fine, but I lean towards following the approach used in the Remix starters and run `build` before `dev`. What do you think?

```suggestion
    "dev": "npm run build && run-p \"dev:*\"",
```

---

Yeah it’s a good question. I like the idea of being close to form. The downsides to this is:
1. Extra processes. Might not be a big deal, but you do basically build the tailwind file and the build files only to the update them
2. If there’s an error building (such as a type error) this will fail. I don’t believe it’s be the case with just dev 

Not big issues, and some might argue 2 is a benefit 🤷🏼‍♂️

Ultimately though I’ll go with whatever you like best

---

Good points! I was just thinking that seeing the error initially isn't pretty, but I don't feel too strongly about it. :) Merged!

---

Also, thanks again for bringing this up in the first place! 💟